plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/CredentialHelper.java

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.appengine.login;
+package com.google.cloud.tools.eclipse.login;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import com.google.api.client.auth.oauth2.Credential;
+import com.google.cloud.tools.eclipse.login.Constants;
```

Comment:
This won't be necessary. Might be the result of a timing issue when auto-generating `Constants.java`. Try reloading projects if you see any problem.